src/project.godot

```diff
@@ -95,6 +95,10 @@ window/size/test_height=540
 window/stretch/mode="2d"
 window/stretch/aspect="expand"
 
+[editor_plugins]
+
+enabled=PoolStringArray(  )
+
 [importer_defaults]
```

Comment:
I try to avoid adding the weird random changes that godot adds to github. This change fyi is because you enabled/disabled a plugin. but it doesn't do anything other than mess with merges

Reply:
Ok. I will try to avoid this in the future.